### ISSUE TITLE
[Bug] Vue 3 in-view animation repeat after leaving and re entering viewport instead of showing it once

### ISSUE THREAD

**Issue by fristyr** [motiondivision/motionone#201]
**1. Describe the bug**
I use vue 3 (with latest Nuxt 3) 
motion: "^10.16.2"

I want to animate component when it enter view port, but instead of animating it once I see animation one more time when I leave viewport of that element and enter it again. And it's repeating. In simple motion without vue, animation fires only once, when I enter viewport 

**2. IMPORTANT: Provide a CodeSandbox reproduction of the bug**

```TypeScript
<Motion
     :initial="{ opacity: 0, x: -50 }"
     :in-view="{
      opacity: 1,
         x: 0,
         transition: {
           delay: 0.5,
           easing: [0.17, 0.55, 0.55, 1],
            duration: 0.9,
           },
         }">
    <p>
       Lorem Ipsum
     </p>
</Motion>
```

**3. Steps to reproduce**

**4. Expected behavior**

Animate only once on viewport 

**5. Video or screenshots**

If applicable, add a video or screenshots to help explain the bug.

**6. Browser details**

**Comment by GabrielZulian** [comment]
Same issue here

**Comment by mattgperry** [comment]
This is now a `wontfix`, see https://github.com/motiondivision/motionone/discussions/241

Hopefully future maintainers will have more time to take a look at this one!
